### PR TITLE
Initial Brights Nights readme

### PR DESCRIPTION
**Pull request by Firestorm01X2**
Initial Brights Nights readme
Go here to preview it:
https://github.com/Firestorm01X2/Cataclysm-BN/tree/bn-readme

#### Summary

 Initial Brights Nights readme

#### Purpose of change

BN needs its own readme.

#### Describe the solution

Inital version of BN readme.

### CODE REVIEW

**Comment by Firestorm01X2** [comment]
I knew to know location of official discord. Or any other official location. I am not sure that they has to be cdda ones.

**Comment by Coolthulhu** [comment]
Something along the lines of "For help, suggestions, questions, check out the BN channel on DDA discord: official-DDA-here".
For the other two (modding and Burning Cyborg Man), I'd say something like "There are also BN channels on the following discords".

I wouldn't name it "official discord". It is official DDA discord, with BN being just a tiny channel.
I'd wait with "official official" BN discord until there is a non-microscopic community around the fork.

File: README.md [hunk 1]
```diff
@@ -1,43 +1,20 @@
...
-[![Language grade: C/C++](https://img.shields.io/lgtm/grade/cpp/g/CleverRaven/Cataclysm-DDA.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/CleverRaven/Cataclysm-DDA/context:cpp)
-[![Open Source Helpers](https://www.codetriage.com/cleverraven/cataclysm-dda/badges/users.svg)](https://www.codetriage.com/cleverraven/cataclysm-dda)
-[![Commit Activity](https://img.shields.io/github/commit-activity/m/CleverRaven/Cataclysm-DDA)](https://github.com/CleverRaven/Cataclysm-DDA/graphs/contributors)
-[![Lines of Code](https://tokei.rs/b1/github/CleverRaven/Cataclysm-DDA?category=code)](https://github.com/XAMPPRocky/tokei)
+While some have described it as a "zombie game", there is far more to Cataclysm than that. Struggle to survive in a harsh, persistent, procedurally generated world. Scavenge the remnants of a dead civilization for food, equipment, or, if you are lucky, a vehicle with a full tank of gas to get you the hell out of Dodge. Fight to defeat or escape from a wide variety of powerful monstrosities, from zombies to giant insects to killer robots and things far stranger and deadlier, and against the others like yourself, who want what you have.
```
Comment (Coolthulhu) [inline README.md:6]: > get you the hell out of Dodge. Fight

Looks like the sentences got melded a bit.

**Comment by Firestorm01X2** [comment]
Ok. Initially done. There is moree in /doc.

**Comment by iillyyaa2033** [comment]
@Firestorm01X2, please recheck Discord invites in README.md - I've got "invite expired" error message for the last two

**Comment by Firestorm01X2** [comment]
> 
> 
> @Firestorm01X2, please recheck Discord invites in README.md - I've got "invite expired" error message for the last two

Got it.

**Pull request merged**